examples/documentation_examples/examples_test.go

```diff
@@ -48,6 +49,16 @@ func TestDocumentationExamples(t *testing.T) {
 	documentation_examples.RunCommandExamples(t, db)
 	documentation_examples.IndexExamples(t, db)
 	documentation_examples.VersionedAPIExamples()
+
+	// Because it uses RunCommand with an apiVersion, the strict count example can only be
+	// run on 5.0+ without auth.
+	ver, err := getServerVersion(ctx, client)
+	auth := os.Getenv("AUTH") == "auth"
+	if err == nil && testutil.CompareVersions(t, ver, "5.0") >= 0 && !auth {
```

Comment:
Instead of checking for `err == nil` here, use `require.NoError(t, err)` on line 56 to make sure errors returned by `getServerVersion` don't result in unintentionally skipping `VersionedAPIStrictCountExample`.

Reply:
Good point; done.